S/Spot_julia/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "Spot_julia"
-version = v"2.12"
+version = v"2.13"
```

Comment:
Sounds like the version number is 2.13.1, no?

Reply:
Indeed... and I also f***ed up with the sha. Will redo.